hack/make/release-deb

```diff
@@ -64,7 +66,7 @@ for dir in contrib/builder/deb/*/; do
 			-k releasedocker --sign builder "${DEBFILE[@]}"
 	fi
 
-	reprepro -v --keepunreferencedfiles \
+	reprepro -v "$options" \
```

Comment:
This either needs to not have quotes, or we need to switch `options` to be an array and use `"${options[@]}"` (or else we'll be passing an empty string argument to `reprepro` which is bound to break).